### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by simranlotey**
Review my code

### CODE REVIEW

File: src/stories/Alert/Alert.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          className={`custom-alert ${severity}`}
+          style={{ borderRadius, backgroundColor }}
+        >
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            viewBox="0 0 512 512"
+            className="error-icons"
+          >
+            <path d="M256 512A256 256 0 1 0 256 0a256 256 0 1 0 0 512zm0-384c13.3 0 24 10.7 24 24V264c0 13.3-10.7 24-24 24s-24-10.7-24-24V152c0-13.3 10.7-24 24-24zM224 352a32 32 0 1 1 64 0 32 32 0 1 1 -64 0z" />
+          </svg>
```
Comment (divyanshugarg36) [inline src/stories/Alert/Alert.jsx:24]: Dont use inline svgs

File: src/stories/Button/OutlinedButton/OutlinedButton.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    <button
+      className={`outlined-custom-button ${size}`}
+      style={{ color: textColor, borderColor: borderColor, borderRadius: borderRadius }}
+    >
+      {children}
+    </button>
+  );
+};
+
+export default OutlinedCustomButton;
```
Comment (divyanshugarg36) [inline src/stories/Button/OutlinedButton/OutlinedButton.jsx:15]: 4 separate components for buttons?
Try making a single with 4 variants

File: src/stories/InputMessages/InputMessages.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        className={`custom-input-messages ${severity}`}
+        style={{ borderRadius, backgroundColor }}
+      >
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 512 512"
+          className="error-icons"
+        >
+          <path d="M256 512A256 256 0 1 0 256 0a256 256 0 1 0 0 512zm0-384c13.3 0 24 10.7 24 24V264c0 13.3-10.7 24-24 24s-24-10.7-24-24V152c0-13.3 10.7-24 24-24zM224 352a32 32 0 1 1 64 0 32 32 0 1 1 -64 0z" />
+        </svg>
```
Comment (divyanshugarg36) [inline src/stories/InputMessages/InputMessages.jsx:22]: No Inline svg

Comment (simranlotey) [reply src/stories/InputMessages/InputMessages.jsx:22]: ok

**Pull request merged**